docs/en/setup/service-agent/java-agent/agent-optional-plugins/Kotlin-Coroutine-plugin.md

```diff
@@ -0,0 +1,30 @@
+# Skywalking with Kotlin coroutine
+This PR provided an auto instrument support plugin for Kotlin coroutine based on context snapshot.
+
+## Description
```

Comment:
Description official document should like

SkyWalking provide tracing context propagation inside thread. In order to support Kotlin Coroutine, we provide this additional plugin.

Reply:
Your description is PR context.